pkgs/tools/misc/iceportal/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "iceportal";
+  version = "2021-11-1";
+
+  src = fetchFromGitHub {
+    owner = "craftamap";
+    repo = "${pname}-api";
```

Comment:
```suggestion
    repo = "iceportal-api";
```

Reply:
Is this actually resolved? Having the variable in repo doesn't seem right

Reply:
Yes variable within there is completely accessible since it's a child.